source/CIGameScene.cpp

```diff
@@ -143,8 +147,9 @@ void GameScene::dispose() {
  */
 void GameScene::reset() {
```

Comment:
Ok so after doing the stuff I worked on today, this reset method is not actually doing anything because the game scene's init method is called whenever a new game is started so we should not really rely on reset